examples/integrations/hf_trainer_api/adaptive.yaml

```diff
@@ -0,0 +1,49 @@
+name: vision transformer adaptive
+debug: false
+environment:
+  environment_variables:
+    - NCCL_DEBUG=INFO
+    # You may need to modify this to match your network configuration.
+    - NCCL_SOCKET_IFNAME=ens,eth,ib
+resources:
+  slots_per_trial: 2
+records_per_epoch: 1000
+searcher:
+  name: adaptive_asha
+  max_length:
+    epochs: 5
```

Comment:
we should switch to batches so asha can be more aggressive with downsampling e.g. have max_rungs of 5 if max_length is >=256 batches

Reply:
Switched to batches, and increased the number of trials.